src/content/docs/en/basics/layouts.mdx

```diff
@@ -144,66 +174,31 @@ const { frontmatter, url } = Astro.props;
 
 ### Markdown Layout Props
 
-A Markdown/MDX layout will have access to the following information via `Astro.props`:
+A Markdown layout will have access to the following information via `Astro.props`:
 
 - **`file`** - The absolute path of this file (e.g. `/home/user/projects/.../file.md`).
 - **`url`** - If it's a page, the URL of the page (e.g. `/en/guides/markdown-content`).
```

Comment:
It seems to me that Markdown Layouts can only be used with pages, so maybe it's not necessary to say "if it's a page":
```suggestion
- **`url`** - The URL of the page (e.g. `/en/guides/markdown-content`).
```
Even if we take into account the page title rather than the section title, I don't see how a layout could be useful anywhere other than for a page. So it seems redundant...

Reply:
Agreed, love it!